### PR TITLE
Feature/page for playlist

### PR DESCRIPTION
**Pull request by Khrapal-Max**
Create view-playlist
added new path 
- path: 'playlists',
      children: [
        { path: 'view-playlist/:id', component: ViewPlaylistComponent }, etc
Update left side menu - added routerLink on view-playlist

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -24,8 +24,7 @@
     "@angular/router": "^12.1.4",
     "@types/jquery": "^3.5.6",
     "eslint-plugin-import": "^2.23.4",
-    "firebase": "^8.8.0",
-    "firebase-admin": "^9.11.0",
+    "firebase": "^8.9.1",
     "fomantic-ui-css": "^2.8.8",
```
Comment (proskurnicha) [inline frontend/package.json]: Why do we need it?

File: frontend/src/app/modules/main/main-menu/main-menu.component.html [hunk 2]
```diff
@@ -34,17 +34,18 @@ <h5 class="name-route">Liked songs</h5>
...
     </button>
   </div>  
 </div>
 
 <div class="container">
   <router-outlet></router-outlet>
-</div>
\ No newline at end of file
+</div>
+<!-- routerLink="playlists/view-playlist" routerLinkActive="active-link" -->
```
Comment (proskurnicha) [inline frontend/src/app/modules/main/main-menu/main-menu.component.html]: Delete a comment

File: frontend/src/app/modules/main/view-playlist/view-playlist.component.ts [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+  play = () => {};
+
+  loadPlaylistSongs() {
+    this._playlistsService
+      .getPlaylistSongs(this._playlistId)
+      .subscribe((songs) => {
+        this.songs = songs;
+        this.totalCountSongs = songs.length;
+        this._totalTimeSongs = songs
```
Comment (proskurnicha) [inline frontend/src/app/modules/main/view-playlist/view-playlist.component.ts:47]: Violates B3.
https://github.com/BinaryStudioAcademy/quality-criteria/blob/production/source/javascript.md

File: frontend/src/app/modules/main/view-playlist/view-playlist.component.ts [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+          .filter((s) => s.duration > 0)
+          .reduce((sum, current) => sum + current.duration, 0);
+        this.hours = Math.floor(this._totalTimeSongs / 3600);
```
Comment (proskurnicha) [inline frontend/src/app/modules/main/view-playlist/view-playlist.component.ts]: 3600 Magic value move to const

**Review by proskurnicha — CHANGES_REQUESTED** [review]

File: frontend/package.json (hunk 1 above)
Comment (Khrapal-Max) [reply frontend/package.json]: reinstall npm, restored

File: frontend/src/app/modules/main/main-menu/main-menu.component.html (hunk 2 above)
Comment (Khrapal-Max) [reply frontend/src/app/modules/main/main-menu/main-menu.component.html]: Done

File: frontend/src/app/modules/main/view-playlist/view-playlist.component.ts (hunk 3 above)
Comment (Khrapal-Max) [reply frontend/src/app/modules/main/view-playlist/view-playlist.component.ts:47]: eslint - Class Property name `totalTimeSongs` must have one leading underscore(s).eslint@typescript-eslint/naming-convention
 this._totalTimeSongs - is private variable

File: frontend/src/app/modules/main/view-playlist/view-playlist.component.ts (hunk 4 above)
Comment (Khrapal-Max) [reply frontend/src/app/modules/main/view-playlist/view-playlist.component.ts]: Done

File: frontend/package.json (hunk 1 above)
Comment (proskurnicha) [reply frontend/package.json]: undo this changes

File: frontend/src/app/modules/main/view-playlist/view-playlist.component.ts (hunk 3 above)
Comment (proskurnicha) [reply frontend/src/app/modules/main/view-playlist/view-playlist.component.ts:47]: ok, will skip it for now

**Pull request merged**